static/app/components/events/eventReplay/index.tsx

```diff
@@ -100,8 +107,36 @@ function EventReplayContent({
     ),
   };
 
+  // don't try to construct the url if we don't have a group
+  const eventIdFromRouter = router.params.eventId;
```

Comment:
nit: this new code here could be extracted to its own component to make this a bit cleaner